charts_common/lib/src/chart/cartesian/axis/scale.dart

```diff
@@ -254,25 +269,11 @@ class RangeBandConfig {
   /// [percentOfStepWidth] is the percentage of the step from 0.0 - 1.0.
   RangeBandConfig.percentOfStep(double percentOfStepWidth)
       : type = RangeBandType.fixedPercentOfStep,
-        size = percentOfStepWidth {
-    assert(percentOfStepWidth >= 0 && percentOfStepWidth <= 1.0);
-  }
-
-  /// Creates a config that assigns the rangeBand according to the stylepack.
-  ///
-  /// <p>Note: renderers can detect this setting and update the percent based on
-  /// the number of series in their preprocess.
-  RangeBandConfig.styleAssignedPercent()
-      : type = RangeBandType.styleAssignedPercentOfStep,
-        size = StyleFactory.style.rangeBandSize;
-
-  /// Creates a config that defines the rangeBand as the stepSize - pixels.
-  ///
-  /// Where fixedPixels() gave you a constant rangBand in pixels, this will give
-  /// you a constant space between rangeBands in pixels.
-  const RangeBandConfig.fixedPixelSpaceBetweenStep(double pixels)
-      : type = RangeBandType.fixedPixelSpaceFromStep,
-        size = pixels;
+        size = percentOfStepWidth,
+        assert(
```

Comment:
One of the lints wanted the assert in the declaration section of the constructor rather than the block body, once I did that, another lint decided they needed to be resorted which `dart fix` then did. 🤣